README.md

```diff
@@ -151,6 +151,30 @@ This workflow is used for validate binary compatibility the current version.
 uses: playframework/.github/.github/workflows/binary-check.yml@v1
 ```
 
+### Mark Pull Request as Ready To Merge
+
+This workflow is used for mark pull request as ready to merge and **should be latest** in the workflows chain.
+
+:warning: For using this workflow don't forget configure `needs` ([GA docs](https://docs.github.com/en/actions/using-workflows/advanced-workflow-features#creating-dependent-jobs)) 
+attribute for make this workflow latest.
+
+**Path**: [`.github/workflows/mark-green.yml`](.github/workflows/mark-green.yml)
+
+**Image**: [Ubuntu 20.04](https://hub.docker.com/layers/ubuntu/library/ubuntu/20.04/images/sha256-57df66b9fc9ce2947e434b4aa02dbe16f6685e20db0c170917d4a1962a5fe6a9?context=explore)
+
+
+**No Parameters**
+
+**How to use**:
+
+```yaml
+needs: # Should be latest
+  - "check-code-style"
+  - "..."
+  - "tests"
+uses: playframework/.github/.github/workflows/mark-green.yml@v1
```

Comment:
Just proposal. May be someone find more good name 🤔 
As another idea is `rtm.yml`.

Reply:
Let use `rtm.yml`, because this says better what the workflow does. `mark-green` is not always correct, there can always be another pull request (besides GHA) which fails so a pull request will not turn green.